sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -52,6 +52,11 @@ object SQLConf {
 
   }
 
+  val WAREHOUSE_PATH = SQLConfigBuilder("spark.sql.warehouse.dir")
+    .doc("The default location for managed databases and tables.")
+    .stringConf
+    .createWithDefault("${system:user.dir}/spark-warehouse")
```

Comment:
@rxin I think default for this should be something else. This will fail things like https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder/58168/